web/client/plugins/SearchByBookmark.jsx

```diff
@@ -186,8 +192,11 @@ export default createPlugin('SearchByBookmark', {
     },
     containers: {
         Search: {
-            menuItem: searchMenuItem,
-            bookmarkConfig: searchByBookmarkConfig
+            // Display bookmark setting only when logged in user is an ADMIN
+            ...(isLoggedIn && isAdminUserSelector && {
```

Comment:
I don't think this will affect the result of the visibility of this item... isLoggedIn and isAdminUserSelector are function, here you are testing their existience